app/internal/shared/shared.go

```diff
@@ -14,7 +14,7 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/mattes/migrate"
-	_ "github.com/mattes/migrate/source/file"
+	_ "github.com/mattes/migrate/source/file" // must have for migrations
```

Comment:
could not import github.com/mattes/migrate/source/file (cannot find package "github.com/mattes/migrate/source/file" in any of:
	/usr/local/go/src/github.com/mattes/migrate/source/file (from $GOROOT)
	/tmp/tmp.UVbATgtbGw/src/github.com/mattes/migrate/source/file (from $GOPATH))